pyproject.toml

```diff
@@ -21,7 +21,12 @@ skip = "migrations"
 name = "redash"
 version = "23.09.0-dev"
 description = "Make Your Company Data Driven. Connect to any data source, easily visualize, dashboard and share your data."
-authors = ["Your Name <you@example.com>"]
+authors = ["Arik Fraimovich <email>"]
+maintainers = [
+    "Justin Clift <email>",
+    "Guido (Sid) Petri <email>",
+    "Eric Radman <email>",
+]
```

Comment:
We can either setup some mailing list for maintainers or have maintainers interested make a PR to add themselves here.

I'm cool with both options with some preference to the first, as otherwise this list will probably become outdated pretty quick.

Reply:
I also would prefer a mailing list, personally

Reply:
I setup maintainers at redash.io. Whoever wants to be on the distribution list, ping me.

Reply:
@arikfr Please add me to that list. :smile:

Reply:
We should add a comment to show how to request membership or unsubscribe

Reply:
Comment added. Hopefully this makes sense but is still concise?

Reply:
The comment is good. At least provides the right direction

Reply:
@arikfr PING

> We should add a comment to show how to request membership or unsubscribe

This ☝️

Reply:
We should probably have a check list of things to do when adding someone as a maintainer:

1. Add on GitHub
2. Add on Discord
3. Add to mailing list

Btw, currently only @guidopetri and @justinclift are on the mailing list. If you want me to add you, send me your email address.